episodes/11/assets/README.md

```diff
@@ -24,41 +24,37 @@ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.1
 ```
 6. Install Cluster API
 ```
-kustomize build github.com/kubernetes-sigs/cluster-api//config?ref=master | kubectl apply -f -
+kustomize build github.com/kubernetes-sigs/cluster-api//config?ref=v0.3.0-rc.2 | kubectl apply -f -
```

Comment:
We just cut this tag today, and I'll do another validation run with this tag in the morning before we start :)